bin/storm.py

```diff
@@ -241,7 +240,6 @@ def jar(jarfile, klass, *args):
             extrajars=[tmpjar, USER_CONF_DIR, STORM_BIN_DIR],
             args=args,
             daemon=False,
```

Comment:
@revans2 can you please take a look and let us know if there is any special reason for this process to be forked? This process being forked affects the return code. If there is no reason for this process to be forked, this fix will correct the return code. Thanks.